Lib/tkinter/ttk.py

```diff
@@ -321,6 +321,8 @@ def _tclobj_to_py(val):
     elif hasattr(val, 'typename'): # some other (single) Tcl object
         val = _convert_stringval(val)
 
+    if isinstance(val, tuple) and len(val) == 0:
+        return ''
```

Comment:
I'm not sure if this can affect existing code.

Reply:
This change is a backport from main. If the code doesn't make sense, it should be changed in main first.